frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -95,19 +96,70 @@ pub fn roll_to_with_ocw(n: u64) {
 	}
 }
 
+pub struct TrimHelpers {
```

Comment:
I'm a fan of keeping things pub for this crate, but we have a rule to keep all pub items documented.